.dialyzer.ignore-warnings

```diff
@@ -132,6 +132,8 @@ ain.Block':put_state/2. The success typing is (#{'__struct__':='Elixir.Blockchai
 }) -> #{'__struct__':='Elixir.Blockchain.Block', 'header':=#{'__struct__':='Elixir.Block.Header', 's
 tate_root':=_, _=>_}, _=>_}
 
+apps/blockchain/lib/blockchain/block.ex:736: Function create_receipt/6 will never be called
```

Comment:
I'm confused by this message. Why does it think that this function will never be called?

Reply:
I believe this warning was fixed by the word we did in `execute_with_validation`